src/coreclr/vm/threads.cpp

```diff
@@ -7683,6 +7683,10 @@ Frame * Thread::NotifyFrameChainOfExceptionUnwind(Frame* pStartFrame, LPVOID pvL
     return pFrame;
 }
 
+#ifndef _MSC_VER
```

Comment:
Why does the non-MSVC need a redeclaration here? Is this because of `__declspec(selectany)` mechanism? If so, it might be appropriate to remove the duplicate definition/declaration pattern and instead follow a simple compiler agnostic `extern` declaration and define the instance in `jitinterface.cpp`.

Reply:
fixed.